controls/numericinput/control-template.md

```diff
@@ -24,106 +24,262 @@ xmlns:telerik="http://schemas.telerik.com/2022/xaml/maui"
-                        <VisualState x:Name="ReadOnly">
-                            <VisualState.Setters>
-                                <Setter Property="BorderColor" Value="Transparent"/>
-                            </VisualState.Setters>
-                        </VisualState>
-                        <VisualState x:Name="Disabled">
-                            <VisualState.Setters>
-                                <Setter Property="BorderColor" Value="#DFDFDF"/>
-                            </VisualState.Setters>
-                        </VisualState>
-                    </VisualStateGroup>
-                </VisualStateManager.VisualStateGroups>
-                <Grid ColumnDefinitions="*, Auto">
-                    <telerik:NumericInputEntry x:Name="PART_Entry"
-                                               Style="{TemplateBinding ActualEntryStyle}"/>
-                    <HorizontalStackLayout Grid.Column="1"
-                                           Spacing="0">
-                        <telerik:NumericInputButton Style="{TemplateBinding ActualDecreaseButtonStyle}"/>
-                        <telerik:NumericInputButton Style="{TemplateBinding ActualIncreaseButtonStyle}"/>
-                    </HorizontalStackLayout>
-                </Grid>
-            </telerik:RadBorder>
-            <telerik:RadBorder x:Name="PART_FocusBorderVisual"
-                               BorderThickness="0, 0, 0, 1" 
-                               CornerRadius="4"
-                               IsEnabled="{TemplateBinding IsEnabled}"
-                               BorderColor="#696969">
-                <VisualStateManager.VisualStateGroups>
-                    <VisualStateGroup x:Name="CommonStates">
-                        <VisualState x:Name="Normal"/>
-                        <VisualState x:Name="Focused">
-                            <VisualState.Setters>
-                                <Setter Property="BorderColor" Value="{StaticResource PrimaryColor}"/>
-                            </VisualState.Setters>
-                        </VisualState>
-                        <VisualState x:Name="ReadOnly">
-                            <VisualState.Setters>
-                                <Setter Property="BorderColor" Value="Transparent"/>
-                            </VisualState.Setters>
-                        </VisualState>
-                        <VisualState x:Name="Disabled">
-                            <VisualState.Setters>
-                                <Setter Property="BorderColor" Value="#DFDFDF"/>
-                            </VisualState.Setters>
-                        </VisualState>
-                    </VisualStateGroup>
-                </VisualStateManager.VisualStateGroups>
-            </telerik:RadBorder>
-        </Grid>
-    </ControlTemplate>
+<Style x:Key="NumericButtonBaseStyle" TargetType="telerik:NumericInputButton">
+    <Setter Property="TextColor" Value="Black", Dark="White" />
```

Comment:
AppThemeBinding is missing?The Value should be
Value={AppThemeBinding White='Black', Dark='White'} 
Test the complete controltemplate in a sample app, this template cannot be compiled

Reply:
updated